docs/getting-started-scp.md

```diff
@@ -0,0 +1,90 @@
+# Installing AWS Service Broker on SAP Cloud Platform
+This guide helps you integrate AWS services into the Cloud Foundry environment on SAP Cloud Platform by using the AWS Service Broker.
+
+For a high-level overview of how this works, take a look at the following image:
+
+![SCP AWS Service Broker integration](/docs/images/scp-aws-service-broker.png)
+
+## Pre-requisites
+
+* Account on [SAP Cloud Platform, Cloud Foundry environment](https://cloudplatform.sap.com/index.html).
+* Account on [AWS](https://aws.amazon.com/free/?awsf.Free%20Tier%20Types=*all&all-free-tier.sort-by=item.additionalFields.SortRank&all-free-tier.sort-order=asc#featured).
+* [Cloud Foundry CLI](https://help.sap.com/viewer/e275296cbb1e4d5886fa38a2a2c78c06/Cloud/en-US/4a8eb630c2734c01a25090c51d48102b.html).
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+## Configuration in your AWS Environment
+
+1. Sign in to your AWS account and in the AWS console, search for Cloudformation.
+3. Choose Create Stack and under Choose a Template, pick Upload a template to Amazon S3.
+4. Save the contents of the [this file](https://github.com/awslabs/aws-servicebroker/blob/master/setup/prerequisites.yaml) to your local environment as a yaml file. In the AWS console, click Choose and upload this file from your local environment.
+5. Choose Next.
+
+Once the Cloudformation stack creation is complete, the DynamoDB table and the IAM user are displayed as follows:
+
+![AWS Cloudformation Stack](/docs/images/aws-cloudformation.png)
+
+## Deploy the AWS Service Broker to SAP Cloud Platform
+
+1. Download the binary zip, aws-sb-cloudfoundry-app.zip, of the AWS Service Broker [here](https://github.com/awslabs/aws-servicebroker/releases).
+2. Unzip the file. Open a terminal and navigate to the AWS Service Broker repository:
+```
+cd aws-sb-cf-cloudfoundry-app
+```
+3. Log in to SAP Cloud Platform in the Cloud Foundry environment:
+```
+cf login
+```
+4. Adapt the following URL to enter your API endpoint:
+https://api.<REGION_TECHNICAL_KEY>.hana.ondemand.com.
+
+Find out your region and the technical key [here](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/350356d1dc314d3199dca15bd2ab9b0e.html#loiof344a57233d34199b2123b9620d0bb41)
+
+5. Enter username and password for your SAP Cloud Platform account.
+6. Edit the `aws-sb-cf-cloudfoundry-app/manifest.yml` file and replace the values within the angular brackets:
```

Comment:
this file is present in the repo https://github.com/awslabs/aws-servicebroker/blob/master/packaging/cloudfoundry/manifest.yml I'd prefer we reference that file so thata updates to it do not need to be done in 2 places.

Reply:
Resolved it.